forte/data/ontology/top.py

```diff
@@ -1140,6 +1011,10 @@ class Payload(Entry):
         ValueError: raised when the modality is not supported.
     """
 
+    payload_idx: int
+    modality_name: str
+    uri: Optional[str]
+
```

Comment:
Right now it's okay to not store `cache` as dataclass attributes, but we will need to remove all the `payload` lists in `DataPack` in future. I think it worths another PR. Maybe you can fix this in #844

Reply:
I think this needs to be done in a whole different PR. Mainly because we want to do the same for Audio, Text and Image entries.

Reply:
Yeah that makes sense.

Reply:
We can make those changes [here](https://github.com/asyml/forte/issues/847)